errors.go

```diff
@@ -254,11 +254,15 @@ func newError(err ResultError, context *jsonContext, value interface{}, locale l
 	err.SetDetails(details)
 	details["field"] = err.Field()
 
+	if err.DescriptionFormat() == "" && d != "" {
```

Comment:
As newError is only used internally, the if can probably omitted as this statement will always be true